### PR TITLE
feat(slack): add slack and remove gitter

### PR DESCRIPTION
**Pull request by matdehaast**
- Add Slack to header
- Add Slack to community page
- Remove mention of Gitter
- Remove mention of the Summit

### CODE REVIEW

**Comment by emschwartz** [comment]
Do we want to use slack for general help as opposed to just working group discussions?

**Comment by matdehaast** [comment]
> Do we want to use slack for general help as opposed to just working group discussions?

I think we should use it for both. Gitter is already not actively engaged with from the contributors. Why not funnel everyone into a single communication channel?

**Comment by matdehaast** [comment]
@emschwartz based on the last call are you happy we go ahead with this?

File: community.html [hunk 1]
```diff
@@ -42,17 +42,9 @@ <h3>IETF</h3>
...
-          <p>Want to chat about Interledger or have a question to ask? Join one of the rooms on Gitter:</p>
-
-          <ul>
-            <li>Project Home: <a href="https://gitter.im/interledger/Lobby">https://gitter.im/interledger/Lobby</a></li>
-            <li>Connector Operators: <a href="https://gitter.im/interledger/connector.land">https://gitter.im/interledger/connector.land</a></li>
-            <li>Ledger Plugin Developers: <a href="https://gitter.im/interledger/ledger-plugins">https://gitter.im/interledger/ledger-plugins</a></li>
-            <li>Java Development: <a href="https://gitter.im/interledger/java">https://gitter.im/interledger/java</a></li>
-            <li>Spec discussions: <a href="https://gitter.im/interledger/rfcs">https://gitter.im/interledger/rfcs</a></li>
-          </ul>
+          <p>Want to chat about Interledger or have a question to ask? <a target="_blank" href="https://communityinviter.com/apps/interledger/interledger-working-groups-slack">Join on Slack</a></p>
```
Comment (emschwartz) [inline community.html]: Can you mention that most of the channels are for working groups focused on specific topics but that people can ask general questions in the #questions-and-answers channel?

**Pull request merged**